src/app/side-bar/dashboard/dashboard.component.html

```diff
@@ -1 +1,42 @@
-<p>dashboard works!</p>
+<app-header></app-header>
+<app-sub-header class="sub-header-component"></app-sub-header>
+<div fxLayout="column" class="dashboard-container">
+  <div fxLayoutAlign="center center" class="width--full">
```

Comment:
please add screen to this view also.

Reply:
I have added the screen to this view..................Thank you